.travis.yml

```diff
@@ -0,0 +1,16 @@
+sudo: required
+
+language: ruby
+
+rvm:
+  - 2.2.2
+
+services:
+  - docker
+
+before_script:
+  - docker run postgres
```

Comment:
No need for this either, they have their own way of running postgres.

Remember that the way that postgres is run is entirely down to the user/environment. We just happen to be using docker, no one else has to.